src/index.html

```diff
@@ -10,23 +10,54 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Airbnb Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search__container">
+
+      <form
+        action="/"
+        method="get"
+        class="search-bar"
+        data-qa="big"
+      >
+
+        <div class="icon-big">
+
+          <input
+            type="text"
+            data-qa="keypress"
+            placeholder="Try “Los Angeles“"
+            class="search__input big"
+          />
+
+        </div>
+
+      </form>
+
+      <form
+        action="/"
+        method="get"
+        class="search-bar"
+        data-qa="small"
```

Comment:
Keep your attributes correctly formatted. The attributes of the form tag should start on new lines with proper indentation to improve readability.